### PR TITLE
Enable debt removals

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/controllers/api/payments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'active_model_serializers'
```
Comment (Xypus) [inline app/controllers/api/payments_controller.rb]: Why is it needed?

Comment (Xypus) [inline app/controllers/api/payments_controller.rb]: Why is this line needed?

File: app/assets/javascripts/modules/balance/views/balance.coffee [hunk 2]
```diff
@@ -1,8 +1,24 @@
 @Lunchiatto.module 'Balance', (Balance, App, Backbone, Marionette, $, _) ->
   Balance.Balance = Marionette.ItemView.extend
+    DELETE_MESSAGE: 'Are you sure? You cannot undo this action'
```
Comment (Xypus) [inline app/assets/javascripts/modules/balance/views/balance.coffee:3]: Not a JS syntax I think.

**Review by Xypus — COMMENTED** [review]
Minor stuff.

File: app/controllers/api/payments_controller.rb (hunk 1 above)
Comment (ghost) [reply app/controllers/api/payments_controller.rb]: Leftover ofc!

**Review by hkopij — APPROVED** [review]
